sql2.php

```diff
@@ -1,11 +1 @@
-if($_POST['submitupdate']){
-        if (empty($updateInputMessage)){
-            echo "type in the message you would like to update";
-        }else if(empty($updateTitle)){ 
-            echo "type in the title you would like to update";
-        } else{ 
-            $stmtupdate = $pdo->prepare("UPDATE notes SET message_text=? WHERE title=?"); 
-            $stmtupdate->execute([$updateInputMessage, $updateTitle]); 
-            echo "Records was updated successfully."; 
-    }
-}
+"UPDATE notes SET message_text=? WHERE title=?" 
```

Comment:
are the  double quotes (")  necessary?

Reply:
they are necessary for using variables in your query once writing php.